src/app/server/Mdns.cpp

```diff
@@ -115,6 +116,12 @@ CHIP_ERROR AdvertiseCommisioning()
 
     auto advertiseParameters = chip::Mdns::CommissionAdvertisingParameters().SetPort(CHIP_PORT).EnableIpV4(true);
 
+    // TODO: this code assumes the device always boots into commissioning mode
+    // fix this once we are able to configure default boot behavior
+    // and when we can determine whether we have been commissioned yet
+    bool notYetCommissioned = true;
+    advertiseParameters.SetCommissioningMode(notYetCommissioned, false);
```

Comment:
2nd bool parameter is not obvious in meaning. Can we update the SetCommisioning mode signature to something that is immediatley readable without referencing header/docs? Generally using enums, like:

```cpp
advertiseParameters.SetCommisioningMode(kNotYetCommisioned, kWhatIsThisParam)
```

Consider also using builder-style code for constants rather than separate lines.